ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```
@@ -186,3 +186,90 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header {
+  border-bottom: none;
+}
+
+.govuk-header__link--homepage:hover,
+.govuk-header__link--homepage:active {
+  border-bottom: $space-1 solid $color-black;
+}
+
+.govuk-header__navigation-item--active a,
+.govuk-header__navigation-item--active a:hover,
+.govuk-header__navigation-item--active a:visited {
+  color: $color-yellow;
+  font-weight: bold !important;
+
+  &:focus {
+    background-color: transparent;
+    box-shadow: none;
+  }
+}
+
+.govuk-header__navigation-item--active .govuk-header__link,
+.govuk-header__navigation-item .govuk-header__link:hover {
+  color: $color-yellow;
+  font-weight: bold;
+  border-bottom: $space-1 solid $color-yellow;
+  text-decoration: none;
+}
+
+.govuk-header__navigation-item .govuk-header__link {
+  font-size: 0.875rem;
```

Can we use `$typography-sm-text-size` here?
```suggestion
  font-size: $typography-sm-text-size;
```